src/codeu/chat/common/ServerInfo.java

```diff
@@ -0,0 +1,29 @@
+/*Class created by Priyanka Agarwal as per code written
```

Comment:
I added some more comments on the top of the class as well as for the functions to add clarity. Please let me know if I can improve this.